mythril/solidnotary/z3utility.py

```diff
@@ -0,0 +1,19 @@
+from z3 import Solver, sat, simplify, is_bool, is_true, is_false
+from copy import deepcopy
+
+def are_satisfiable(constraints):
+    s = Solver()
+    for c in constraints:
+        s.add(c)
+    return s.check() == sat
+
+def simplify_constraints(constraints): # Todo simplification of the sum of constraints
+    simp_const = []
+    for const in constraints:
+        simp_const.append(simplify(const))
```

Comment:
a list comprehensive would be quick here,
just 
simp_const = [simplify(const] for const in constraints] 
or something like that

Reply:
sorry, these are changes from my master branch, I must have done something horribly wrong when merging in your main master or when pushing. I will figure out how to revert to the last accetable point.

Reply:
After reverting can you merge your branch with the current master of mythril else build might fail